src/Icon/Icon.js

```diff
@@ -34,6 +34,9 @@ export type Color = 'inherit' | 'accent' | 'action' | 'contrast' | 'disabled' |
 
 type ProvidedProps = {
   classes: Object,
+  /**
+   * @ignore
+   */
   theme?: Object,
```

Comment:
I have been wondering, why to we put the theme here in the first place? I have noticed we can remove them with no flow issue.

Reply:
If we can remove them now with @rsolomon's latest PR to react-flow-types, we should do that.